### DISCUSSION TITLE
Different title and body messages for alerting and relax

### DISCUSSION THREAD

**Discussion by castillo92** [grafana/grafana#39600] [Alerting]
Hello,

**Feature Request:**

Be able create more flexible and customized alerts notifications allowing changing the title and body of the alerts

Now, we have te state: [Alerting] and [OK] and a common title and body message.

- The first request is be able to change the string of this states.
- The second is be able to change the title and body for each state.

Thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
